mu4/domain/notation/internal/notationcommander.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NOTATIONCOMMANDER_H
+#define NOTATIONCOMMANDER_H
+
+#include "inotationcommander.h"
+
+#include "modularity/ioc.h"
+#include "context/iglobalcontext.h"
+#include "igetscore.h"
+
+namespace mu::domain::notation {
+class NotationCommander : public INotationCommander
+{
+    INJECT(notation, mu::context::IGlobalContext, context)
```

Comment:
Why is there a dependence on context?

Reply:
I just forget it, ty
It was difficult to notice due to the number of changed files